### ISSUE TITLE
[BUG] Transformations fail for rectangles and circles.

### ISSUE THREAD

**Issue by Nehemek** [TurtleP/LovePotion#124]
When using transformations such as love.graphics.translate(), they will not apply to circles or rectangles if drawn right after calling the functions, however they will work if an image or a canvas has been drawn before them.

Doesn't work:
```
love.graphics.origin()
love.graphics.translate(50,50)
love.graphics.circle("fill",0,0,16)
```

Works:
```
love.graphics.origin()
love.graphics.translate(50,50)
love.graphics.draw(some_image,0,0,0,1,1,0,0,0,0)
love.graphics.circle("fill",0,0,16)
```

System: 3ds v11.14.0-46U
CFW: Luma v10.2.1
Method of execution: Homebrew App
LÖVE Potion version: 2.0.0

**Comment by TurtleP** [comment]
is it just the rectangles and circles?

**Comment by TurtleP** [comment]
in the code it seems polygons should be fine, but circles, lines, ellipses, and arcs do not get the proper code(?)

**Comment by Nehemek** [comment]
I have only tested circle and rectangles. I will test other shapes and elements (like text) to confirm and will get back to you.

**Comment by TurtleP** [comment]
this was fixed in commit d638726 -- some other things will also need fixing (lines, arcs, and polygons it seems)
